src/client/UserSettingModal.ts

```diff
@@ -8,6 +8,88 @@ import "./components/baseComponents/setting/SettingNumber";
 import "./components/baseComponents/setting/SettingSlider";
 import "./components/baseComponents/setting/SettingToggle";
 
+interface KeybindGroup {
+  title: string;
+  binds: {
+    action: string;
+    label: string;
+    description: string;
+    defaultKey: string;
+  }[];
+}
+
+const keybindGroups: KeybindGroup[] = [
+  {
+    title: "user_setting.view_options",
+    binds: [
```

Comment:
Hope you don't mind, I refactored the keybind settings here so they're easier to maintain (mainly for the UI component)